<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wise</class>
 <widget class="QMainWindow" name="wise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wise</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="wise_tab" name="tab"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open_PDF_Files"/>
    <addaction name="separator"/>
    <addaction name="action_Close_Page"/>
    <addaction name="separator"/>
    <addaction name="action_Print"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Screen_Mode"/>
    <addaction name="separator"/>
    <addaction name="action_Recent_Files"/>
    <addaction name="separator"/>
    <addaction name="action_Tool_Bar"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Settings"/>
   </widget>
   <widget class="QMenu" name="menu_Pages">
    <property name="title">
     <string>&amp;Pages</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="separator"/>
    <addaction name="action_Release_Notes"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Pages"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="tool_bar">
   <property name="windowTitle">
    <string>Wise: Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit Wise</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Open_PDF_Files">
   <property name="text">
    <string>&amp;Open PDF File(s)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Screen_Mode">
   <property name="text">
    <string>&amp;Full Screen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="action_Close_Page">
   <property name="text">
    <string>&amp;Close Page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="../Icons/icons.qrc">
     <normaloff>:/wise.png</normaloff>:/wise.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="action_Release_Notes">
   <property name="text">
    <string>&amp;Release Notes</string>
   </property>
  </action>
  <action name="action_Left_Panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Left Panel</string>
   </property>
  </action>
  <action name="action_Recent_Files">
   <property name="icon">
    <iconset resource="../Icons/icons.qrc">
     <normaloff>:/recent.svg</normaloff>:/recent.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Recent Files</string>
   </property>
  </action>
  <action name="action_Tool_Bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Tool Bar</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wise_tab</class>
   <extends>QTabWidget</extends>
   <header>wise-tab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tab</tabstop>
 </tabstops>
 <resources>
  <include location="../Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
